websites.yml

```diff
@@ -56,5 +61,7 @@ avatar:
     key: image_xlarge_url
   soundcloud: opengraph
   tumblr: opengraph
+  opensuse:
+    key: profile_icon_wrapper
```

Comment:
In it's current state, `key` is used for JS style `key: value` searching. You'll have to make a new one.

Reply:
If this is working, then why create another one?

Reply:
@jayvdb: I don’t think it’s working? Is it? I didn’t see this key on the page..